hapi/src/services/state-history-plugin.service.js

```diff
@@ -149,38 +149,6 @@ const cleanOldBlocks = async () => {
   await hasuraUtil.request(mutation, { date })
 }
 
-const worker = async () => {
```

Comment:
@Torresmorah  are you sure this wasnt being used for missed blocks feature?

Reply:
Yes, I had added it to get the block via get_block, but the plugin does that function